### PR TITLE
Fire data app v1.0

### PR DESCRIPTION
**Pull request by Gopxfs**
# Summary 📒 
- Created an application that gathers data from the [INPE API](https://queimadas.dgi.inpe.br/queimadas/dados-abertos/#), corresponding to the number of fires in South America in the last 24 hours.
- On the homepage, it is displayed the total number of fires as well as the number filtered by each country.
- The user can select any country to show the percentage that country is contributing to the total, and also a list of each individual state's contribution.
- It is also possible to sort the fire quantity by higher/lower first.
- _Additional:_ Added about page and desktop version.

## Tools used 🔨 
- React (React router, life cycle, testing library...)
- Redux (Redux toolkit,  slices, createAsyncThunk...)
- Used store, actions & reducers in React.
- SCSS for styling.
- Jest Library.
- Used [React&Redux linters](https://github.com/microverseinc/linters-config/tree/master/react-redux).
- Followed React best practices.

## Requirements ✅ 
- Home page
  - When the page loads, the app shows the list of categories filtered out by country.
  - Along with the country name, the number of fires is displayed.
  - When the user clicks on a category, the application navigates to the details page (states list).
- Details page (states list).
  - Retrieves data from the API to show detailed data in the category.
  - When the user clicks on the back button, navigates to the homepage.
### Technical requirements 📚 
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- The data that is retrieved from the API is filtered using a Filter stateless component.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is deployed and accessible online. [Check here](https://fire-data.herokuapp.com/).
### Documentation 🎥 
- Present the features of the project with a video. [Check README](https://github.com/Gopxfs/fire-data-app/blob/dev/README.md).
### Additional requirements 💯 
- Added effects to flags/navbar.
- Implemented an additional page (about).
- Added a good desktop design.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,4 +36,4 @@ To edit this project:
 Give a :star: if you like my project!
```
Comment (Ismailco) [inline README.md:40]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/), kindly add this to your README

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @Gopxfs, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/pages/CountryPage.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import StateList from '../components/StateList';
+import CountryData from '../components/CountryData';
+import { closeHamburger } from '../redux/FunctionalitySlice';
+
+const CountryPage = () => {
+  const dispatch = useDispatch();
+  return (
+    <>
+      <div className="backToHomepage">
+        <Link to="/" onClick={() => dispatch(closeHamburger())}>Back to homepage</Link>
```
Comment (Ismailco) [inline src/pages/CountryPage.js:12]: - [ ] The filter functionality is not working on the individual (country) pages, kindly fix this so it works as intended. I guess the issue is that you're not generating a unique id (key) for each city

- The console error:
Screenshot_20220728_075409

**Review by Ismailco — CHANGES_REQUESTED** [review]

File: src/pages/CountryPage.js (hunk 2 above)
Comment (Gopxfs) [reply src/pages/CountryPage.js:12]: Hey, @Ismailco Can you specify this? I couldn't reproduce the error.

Comment (Ismailco) [reply src/pages/CountryPage.js:12]: @Gopxfs, I checked again and it's working normally, I don't understand what happened, but sometimes the same key is used for different cities, in this case, it breaks the filter function

**Review by medaminedev66 — APPROVED** [review]
Hi @Gopxfs 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- Integration tests have been added
- Your design looks great
- Your code organization was really good
- Your app works as expected and runs without errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**